lib/grandpa/message_handler_test.go

```diff
@@ -553,7 +577,7 @@ func TestMessageHandler_HandleCatchUpResponse(t *testing.T) {
 
 	h := NewMessageHandler(gs, st.Block, telemetryMock)
 
-	round := uint64(77)
+	round := uint64(1)
```

Comment:
Why was it `77` before?

Reply:
So you catch up only when you are more than 2 block behind some other peer. There is not anything special about 77, it could be anything more than 3. I changed this because block number 77 is not in the block state, but I might have to change this piece code for address elizabeth's comment.